Dockerfile

```diff
@@ -1,8 +1,18 @@
+FROM maven:3.6.0-jdk-11-slim AS build
```

Comment:
@mcruzdev why did you changed this? any reason to do that? Multi staged build are well known to cause some issues with pipelines

Reply:
I did because was the way that i found to pass a jdbc driver to project disacoupled.